src/main/java/com/thoughtworks/pacman/ui/screens/IntroScreen.java

```diff
@@ -1,28 +1,130 @@
 package com.thoughtworks.pacman.ui.screens;
 
-import java.awt.Dimension;
+import java.awt.*;
 import java.awt.Graphics2D;
 import java.awt.Image;
-import java.awt.event.KeyEvent;
+import java.awt.event.*;
+import java.awt.Color;
+import java.awt.Rectangle;
+
+import java.awt.font.TextLayout;
+
 
 import com.thoughtworks.pacman.core.Game;
 import com.thoughtworks.pacman.ui.ImageLoader;
 import com.thoughtworks.pacman.ui.Screen;
 
+
 public class IntroScreen implements Screen {
-    static final Image TITLE_SCREEN_IMAGE = ImageLoader.loadImage(Screen.class, "titleScreen.jpg");
+    static final Image TITLE_SCREEN_IMAGE = ImageLoader.loadImage(Screen.class, "deletedText.jpg");
```

Comment:
Yeni dosya yaratmak yerine neden varolan dosyaları güncellemiyorsun? Örneğin titleScreen.jpg artık kullanılmayan bir resim olacağına göre, doğrudan overwrite ederek yeni resmi de titleScreen.jpg diye isimlendirmek daha mantıklı değil mi?